example_org/policy/wipeout/default.yaml

```diff
@@ -0,0 +1,17 @@
+plan_id: policy/wipeout/default
```

Comment:
Did we agree to use path as the id?

Reply:
It's part of the design doc. Using path as the id is recommened.